syntax = "proto3";

package NAlice.NScenarios;

option go_package = "a.yandex-team.ru/alice/megamind/protos/scenarios";
option java_package = "ru.yandex.alice.megamind.protos.scenarios";
option java_outer_classname = "ScenarioRequestMeta";

message TRequestMeta {
    string ContentType = 1;

    reserved 2; reserved "RTLogToken";
    reserved 3; reserved "Scenario";
    reserved 4; reserved "Action";

    uint64 RandomSeed = 5;
    string OAuthToken = 6;
    string Lang = 7;
    string RequestId = 8;
    string UserLang = 9;
    string UserTicket = 10;
    string ClientIP = 11;
}
